File: 02_javaScript/04_UserInput/userInput.js
/*
-----------------------------------------
Topic : User Input
Language : JavaScript
Bootcamp : Data Analytics Bootcamp
Author : Shubham Waghmare
-----------------------------------------
*/

// ==========================
// String Input
// ==========================

let name = prompt("Enter your name:");

console.log("Hello,", name);

// ==========================
// Number Input
// ==========================

let age = Number(prompt("Enter your age:"));

console.log("Age:", age);

// ==========================
// Float Input
// ==========================

let height = parseFloat(prompt("Enter your height (in feet):"));

console.log("Height:", height);

// ==========================
// Boolean Input
// ==========================

let isStudent = prompt("Are you a student? (true/false)");

isStudent = isStudent.toLowerCase() === "true";

console.log("Student:", isStudent);

// ==========================
// Multiple Inputs
// ==========================

let city = prompt("Enter your city:");
let course = prompt("Enter your course:");

console.log("----- Student Information -----");

console.log("Name:", name);
console.log("Age:", age);
console.log("Height:", height);
console.log("Student:", isStudent);
console.log("City:", city);
console.log("Course:", course);

File: 01_Python/04_UserInput/userinput.py
"""
-----------------------------------------
Topic : User Input
Language : Python
Bootcamp : Data Analytics Bootcamp
Author : Shubham Waghmare
-----------------------------------------
"""

# ==========================
# Taking String Input
# ==========================

name = input("Enter your name: ")

print("Hello,", name)

# ==========================
# Taking Integer Input
# ==========================

age = int(input("Enter your age: "))

print("Your age is:", age)

# ==========================
# Taking Float Input
# ==========================

height = float(input("Enter your height (in feet): "))

print("Your height is:", height)

# ==========================
# Taking Boolean Input
# ==========================

is_student = input("Are you a student? (True/False): ")

is_student = is_student.lower() == "true"

print("Student:", is_student)

# ==========================
# Multiple Inputs
# ==========================

city = input("Enter your city: ")
course = input("Enter your course: ")

print("\n----- Student Information -----")
print("Name :", name)
print("Age :", age)
print("Height :", height)
print("Student :", is_student)
print("City :", city)
print("Course :", course)